.github/workflows/papers-and-specs.yml

```diff
@@ -42,8 +49,16 @@ jobs:
           done 
 
       - name: Publish Papers
+        if: github.event_name == 'push' || github.event_name == 'workflow_dispatch'
         uses: JamesIves/github-pages-deploy-action@v4.7.3
         with:
           folder: _resources
           target-folder: resources
-          single-commit: true
\ No newline at end of file
+          single-commit: true
+
+      - name: Publish Preview Papers 
```

Comment:
Is this what enables the preview of the spec? I'm not seeing any mention of `plutus-core-spec` here.